TileFormats/Instanced3DModel/README.md

```diff
@@ -22,61 +22,129 @@ A tile is composed of a header section immediately followed by a body section.
 
 **Figure 1**: Instanced 3D Model layout (dashes indicate optional fields).
 
-![](figures/layout.png)
+![header layout](figures/header-layout.png)
+
+![body layout](figures/body-layout.png)
 
 ## Header
 
-The 28-byte header contains the following fields:
+The 76-byte header contains the following fields:
 
-|Field name|Data type|Description|
-|----------|---------|-----------|
+| Field name | Data type | Description |
+| --- | --- | --- |
 | `magic` | 4-byte ANSI string | `"i3dm"`.  This can be used to identify the arraybuffer as an Instanced 3D Model tile. |
 | `version` | `uint32` | The version of the Instanced 3D Model format. It is currently `1`. |
 | `byteLength` | `uint32` | The length of the entire tile, including the header, in bytes. |
-| `batchTableByteLength` | `uint32` | The length of the batch table in bytes. Zero indicates there is not a batch table. |
+| `featureTableByteLength` | `uint32` | The length of the feature table in bytes. |
```

Comment:
For feature and batch tables, don't we need to store both the JSON length and binary length like @lilleyse did for `pnts`?